diff --git a/setup.py b/setup.py
index 7d68954..0c211c4 100644
--- a/setup.py
+++ b/setup.py
@@ -113,14 +113,14 @@ setup(
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'requests[security]',
+        'requests',
         'tldextract',
         'future',
         'cryptography',
         'pyyaml',
     ],
 
-    extras_require=extras_require,
+    #extras_require=extras_require,
 
     # To provide executable scripts, use entry points in preference to the
     # "scripts" keyword. Entry points provide cross-platform support and allow
